<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Guitar app</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/icon.png</normaloff>Images/icon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-color: rgb(0, 255, 255);
height: 50px;
width: 90px;
border:3px solid rgb(0,0,0);
border-radius:8px;
padding:10px;
font-family: Helvetica;
font-weight: bold;
font-variant: normal;
font-size: 15px;
}

QPushButton:hover{
	color: rgb(247, 178, 57);
	font-size: 18px;
}
</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>177</y>
      <width>771</width>
      <height>466</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="8" column="2">
      <widget class="Menu_button" name="Dis_Button">
       <property name="text">
        <string>Dis</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="Menu_button" name="Metronome_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Metronome</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="Menu_button" name="Cis_Button">
       <property name="text">
        <string>Cis</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Menu_button" name="D_Button">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="Menu_button" name="E_Button">
       <property name="text">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="Menu_button" name="B_Button">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Menu_button" name="Quiz_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
&quot;color: rgb(255, 255, 255)&quot;
}</string>
       </property>
       <property name="text">
        <string>Quiz</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="Menu_button" name="F_Button">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="Menu_button" name="Tuner_Button">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Tuner</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="Menu_button" name="A_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="Menu_button" name="C_Button">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="Menu_button" name="G_Button">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="Menu_button" name="Fis_Button">
       <property name="text">
        <string>Fis</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="Menu_button" name="Gis_Button">
       <property name="text">
        <string>Gis</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="Menu_button" name="H_Button">
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>700</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>Images/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Menu_button</class>
   <extends>QPushButton</extends>
   <header>Menu_button</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Qrc_files/main_background.qrc"/>
 </resources>
 <connections/>
</ui>
